pkg/upgrade/upgrade.go

```diff
@@ -182,13 +185,18 @@ func (u *Upgrader) Version() semver.Version {
 	return u.boot.Version()
 }
 
+// nextUpdate returns the interval until the next update
+// which is approximately 60 minutes + jitter interval(0-10 minutes)
+// to make sure not all nodes run upgrader at the same time
 func (u *Upgrader) nextUpdate() time.Duration {
 	jitter := rand.Intn(checkJitter)
 	next := checkForUpdateEvery + (time.Duration(jitter) * time.Minute)
 	log.Info().Str("after", next.String()).Msg("checking for update after")
 	return next
 }
 
+// remote creates a new taglink for the latest version of the
+// used flist that matches the same run mode (ex: development)
```

Comment:
this method actually only finds the tag link associated with the network. Then this taglink is later
resolved to the list of packages that we need to install
```suggestion
// remote finds the `tag link` associated with the node network (for example devnet)
```